recipe/build.sh

```diff
@@ -2,6 +2,13 @@
 
 export LDFLAGS="-L${PREFIX}/lib"
 export CFLAGS="${CFLAGS} -O3 -I${PREFIX}/include"
+
+if [ "$(uname)" == "Darwin" ];
+then
+    export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
+    brew uninstall gmp
```

Comment:
Will a regular conda user have the LD_LIBRARY_PATH set?
On Mar 28, 2016 10:09, "jakirkham" <notifications@github.com> wrote:

> In recipe/build.sh
> <https://github.com/conda-forge/glpk-feedstock/pull/4#discussion_r57597770>
> :
>
> > @@ -2,6 +2,13 @@
> >
> >  export LDFLAGS="-L${PREFIX}/lib"
> >  export CFLAGS="${CFLAGS} -O3 -I${PREFIX}/include"
> > +
> > +if [ "$(uname)" == "Darwin" ];
> > +then
> > +    export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
> > +    brew uninstall gmp
>
> Could we check that we are on Travis CI (e.g. TRAVIS
> <https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables>
> )?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/glpk-feedstock/pull/4/files/791ec8c8a882e323935bcc80e7a7800c89f81577#r57597770>
>